src/pages/blog-post-template-page/BlogPostTemplatePage.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import * as Bs from 'react-bootstrap';
+import PageWrapper from '../../components/page-wrapper/PageWrapper';
+import { Col, Container, Row } from 'react-bootstrap';
+import { BlogPostTemplatePageStrapiContent } from '../../data/interfaces/blog-post-template-page/BlogPostTemplatePageStrapiContent';
+import ReactMarkdown from 'react-markdown';
+
+type Props = {
+  strapiData: BlogPostTemplatePageStrapiContent;
+};
+
+const BlogPostTemplatePage: React.FC<Props> = ({ strapiData }) => {
+  const centerImageStyle = {
```

Comment:
This can be solved using bootstrap utility classes:

`d-flex justify-content-center `should do this. There are lots of examples in the codebase. [DOCS](https://getbootstrap.com/docs/5.3/utilities/flex/#justify-content)